File: Latihan_B/php/Processor.php
<?php
class Processor {
    private $name;
    private $price;

    // constructor
    function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }

    // method set dan get untuk name dan price
    function setName($name) {
        $this->name = $name;
    }
    function setPrice($price) {
        $this->price = $price;
    }
    
    function getName() {
        return $this->name;
    }
    function getPrice() {
        return $this->price;
    }

    // method print
    function printProcie() {
        echo "  Processor". "<br/>";
        echo "Name        : ".  $this->getName(). "<br/>";
        echo "Price       : ". "Rp. ". $this->getPrice(). "<br/>";
        echo "************************************". "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>

File: Latihan_B/python/Processor.py
class Processor:
    # atribut private
    __name = ""
    __price = 0

    # constructor
    def __init__(self, name="", price=0):
        self.__name = name
        self.__price = price

    # method set dan get untuk name dan price
    def setName(self, name):
        self.__name = name
    def setPrice(self, price):
        self.__price = price

    def getName(self):
        return self.__name
    def getPrice(self):
        return self.__price
    
    # method print
    def printProcie(self):
        print(" Processor")
        print("Name        : " + self.getName())
        print("Price       : " + "Rp. " + str(self.getPrice()))
        print("*********************************")
        